mm2src/mm2_bitcoin/serialization/src/reader.rs

```diff
@@ -74,15 +76,17 @@ impl<'a> Reader<&'a [u8]> {
             buffer,
             peeked: None,
             coin_variant: CoinVariant::Standard,
+            coin_name: None,
```

Comment:
`coin_name` actually duplicates the meaning of `coin_variant`. Please extend `enum CoinVariant` instead.